src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -10066,6 +10070,23 @@ instruct CallLeafDirect(method meth, rFlagsReg cr)
   ins_pipe(pipe_class_call);
 %}
 
+// Call Runtime Instruction without safepoint and with vector arguments
+
+instruct CallLeafDirectVector(method meth, rFlagsReg cr)
+%{
+  match(CallLeafVector);
+
+  effect(USE meth, KILL cr);
```

Comment:
One more question here. I didn't check the details of `CallLeafVector`. Is it safe to assume that `FRM` will be saved and restored before and after the runtime call? Check this: https://bugs.openjdk.org/browse/JDK-8330094

Reply:
Good question!
Let me do some further investigation and get back later.

Reply:
I see sleef code only set frm to RNE, but I'm not quite sure.
Even if we can make sure current sleef only set frm to RNE, seems to me we can not depends on current implement, it could change. Although good news is we don't update sleef regularly.
Maybe we should take similar action as call-to-java and return-from-jni?

Reply:
Sounds like a reasonable solution!
Anyone has other thoughts please kindly let me know.

Reply:
added some code to check `frm` after sleef calls.